code/datums/map_config.dm

```diff
@@ -67,9 +69,7 @@
 		/datum/equipment_preset/survivor/engineer
 	)
 
-	synth_survivor_types = list(
-		/datum/equipment_preset/synth/survivor
-	)
+	synth_survivor_types = typesof(/datum/equipment_preset/synth/survivor)
```

Comment:
You don't want to pick up the FORECON synthvivor (or other future map specific synthvivors) so you'll have to be explicit with your list.